pkg/domain/infra/abi/network.go

```diff
@@ -164,9 +164,14 @@ func (ic *ContainerEngine) NetworkRm(ctx context.Context, namesOrIds []string, o
 				}
 			}
 		}
+		net, err := ic.Libpod.Network().NetworkInspect(name)
```

Comment:
@Luap99 
Based on the  test failure logs, I suspect this code might be affecting existing test cases, though I could be mistaken.

Do you have any suggestions on how to retrieve network details without using the NetworkInspect function to avoid unnecessary test case failures?

Reply:
yes I think the current code seems find to work around that, ignoring define.ErrNoSuchNetwork seems correct.

Long term it might be better to move these events into c/common/libnetwork but that is much more work and not needed here.